src/wmnds/patterns/header-v2/_header-v2.njk

```diff
@@ -0,0 +1,163 @@
+                          <use xlink:href="#wmnds-general-star" href="#wmnds-general-star"></use>
+                        </svg> {{subnavItem.name}}</a>
+                        {%- if subnavItem.subnavItems %}
+                      <button class="wmnds-mega-menu__collapse-toggle" aria-expanded="false" aria-label="Toggle {{subnavItem.name}} menu" aria-controls="{{params.id}}-submenu-child-menu"><svg class="wmnds-mega-menu__link-arrow-icon"><use xlink-href="#wmnds-general-chevron-right" href="#wmnds-general-chevron-right"></use></svg></button>
+                      <ul id="{{params.id}}-submenu-child-menu-{{ subnavItem.name | lower | replace(" ", "-") }}-{{ loop.index }}" class="wmnds-mega-menu__sub-menu-child-menu">
+                        {% for subnavchildItem in subnavItem.subnavItems -%}
+                        <li class="wmnds-mega-menu__sub-menu-child-item">
+                          <a href="#" class="wmnds-mega-menu__sub-menu-child-link">{{subnavchildItem.name}}</a>
+                        </li>
+                        {%- endfor %}
+                      </ul>
+                      {%- endif %}
+                    </li>
+                    {%- endfor %}
+                  </ul>
+                </div>
+              </div>
+              {%- endif %}
+            </li>
+          {%- endfor %}
+          </ul>
+        </div>
+        <button class="wmnds-mega-menu__primary-menu-link wmnds-mega-menu__search">
+          <svg class="wmnds-mega-menu__search-icon"><use xlink:href="#wmnds-general-search" href="#wmnds-general-search"></use></svg>
+        </button>
+      </nav>
+      {%- else %}
+      {#- Else with a navigation -#}
+      <!-- Nav items -->
+      <nav class="wmnds-header__vertical-align wmnds-col-auto" aria-label="{{navigationAriaLabel}}">
+        <ul class="wmnds-header__links wmnds-header__main-links">
+          {% for navItem in navItems -%}
+          <li class="wmnds-header__link">
+            <a href="/{{ navItem.name | lower | replace(" ", "-") }}/" title="{{ navItem.name }} page" target="_self" class="wmnds-link" >
+              {{ navItem.name}}
+            </a>
+          </li>
+          {%- endfor %}
+        </ul>
+        {%- if params.app %}
+        <!-- Back button -->
+        <ul class="wmnds-header__links wmnds-header__back-btn">
+          <li class="wmnds-header__link">
+            <a
+              href="#"
+              title="Go back to previous page"
+              target="_self"
+              class="wmnds-link"
+              onClick="goBack(event);"
+            >
+              Back
+            </a>
+          </li>
+        </ul>
+        <script>
+          // Function that sends user to previous page they were on,
+          // This can be included in your local javascript file
+          function goBack(e) {
+            e.preventDefault();
+            window.history.back();
+          }
+        </script>
```

Comment:
We can remove the old header from the iffy statement as we should be using the new one going forward. 
The older one was just a temporary solution until this came along.